enclave-runner/src/lib.rs

```diff
@@ -30,3 +30,4 @@ mod usercalls;
 pub use command::Command;
 pub use library::Library;
 pub use loader::{EnclaveBuilder, EnclavePanic};
+pub use usercalls::{UsercallExtension, SyncStream};
```

Comment:
I think this makes more sense in its own module. Can you make `mod usercalls` `pub`?